cloudferrylib/os/actions/check_cloud.py

```diff
@@ -17,8 +17,10 @@
 from keystoneclient.openstack.common.apiclient import exceptions as ks_exc
 from novaclient import exceptions as nova_exc
 from cloudferrylib.base import exception
+import time
 
 LOG = utl.get_log(__name__)
+LIMIT_RETRY_CHECK_INSTANCE = 25
```

Comment:
Maybe it is better to use `retry` option from `migrate` section of config instead of hardcoded value?